app/controllers/users_controller.rb

```diff
@@ -69,4 +81,11 @@ def reset_password
   def get_params
     params.require(:user).permit!

```

Comment:
This is very dangerous, I would not recommend ever using `permit!` unless it fits a very specific use-case.